src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -44,6 +44,22 @@ public static string GetFullPath(string path)
             return result;
         }
 
+        public static bool TryGetFullPath(ReadOnlySpan<char> path, Span<char> destination, out int charsWritten)
+        {
+            charsWritten = 0;
+
+            if (path.Length == 0 || path.IndexOf('\0') != -1)
```

Comment:
This return false for both error conditions that won't ever go away; and for buffer-too-small condition.

How do we expect this API to be used? Does the caller have to have hard coded limit on maximum buffer size to try until it declares that it won't work anyway? It does not feel right.

Reply:
@JeremyKuhne

Reply:
@JeremyKuhne can you help this one along